src/main/kotlin/com/fasterxml/jackson/module/kotlin/KotlinMixins.kt

```diff
@@ -2,8 +2,10 @@ package com.fasterxml.jackson.module.kotlin
 
 import com.fasterxml.jackson.annotation.JsonCreator
 import com.fasterxml.jackson.annotation.JsonIgnore
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
 import com.fasterxml.jackson.annotation.JsonProperty
 
+@JsonIgnoreProperties("endExclusive")
```

Comment:
I think it'd be more consistent to add ignoral to a method, considering this is done for the other accessors.

Then again there's already #583 for that.